app.py

```diff
@@ -6,7 +6,32 @@
 
```

Comment:
You don't really need that submodule `DevOps-Course-Starter` listed above

Reply:
how do I remove it?